wherehows-web/app/routes/application.js

```diff
@@ -6,7 +6,8 @@ const {
   Route,
   run,
   get,
-  inject: { service }
+  inject: { service },
+  testing,
```

Comment:
Please use `import config from 'trex-web/config/environment';` instead of importing `testing`.

```
import config from 'trex-web/config/environment';
...

if (config.environment !== 'test') {
  ...
}

```